test/e2e/test/version.go

```diff
@@ -31,6 +31,21 @@ func SkipInvalidUpgrade(t *testing.T, srcVersion string, dstVersion string) {
 	if !isValid {
 		t.SkipNow()
 	}
+
+	// Temporarily skip upgrades to `8.9.0-SNAPSHOT`
+	dstVer := version.MustParse(dstVersion)
+	if dstVer.Major == 8 && dstVer.Minor == 9 && isSnapshot(dstVer) {
```

Comment:
Reading this, I thought why not just `if dstVersion == "8.9.0-SNAPSHOT"` :)

Reply:
Because I thought it was cleaner to verify that it was an "snapshot" version by parsing the string, and then using `pre.VersionStr`. Agreed it may feel a bit "over engineered" though 😄